napalm_ros/ros.py

```diff
@@ -370,7 +371,13 @@ def get_config(self, retrieve='all', full=False, sanitized=False):
             command.append("show-sensitive")
         if version.major <= 6 and sanitized:
             command.append("hide-sensitive")
-        self.ssh.connect(self.hostname, port=self.ssh_port, username=self.username, password=self.password)
+        self.ssh.connect(
+            self.hostname,
+            port=self.ssh_port,
+            username=self.username,
+            password=self.password,
+            look_for_keys=self.paramiko_look_for_keys
```

Comment:
```suggestion
            look_for_keys=self.paramiko_look_for_keys,
```
missing `,` at the end.